src/python/scripts/twitter_scrooge_pluck_cmd.py

```diff
@@ -0,0 +1,49 @@
+import os
+import re
+import sys
+
+def parse_gen_file_map(filename):
+  r = re.compile("(\S+) -> (\S+)")
+  with open(filename) as file:
+    m = {}
+    for line in file:
+      g = r.match(line)
+      k = g.group(1)
+      v = m.get(k, set())
+      v.add(g.group(2))
+      m[k] = v
+  return m
+
+def main(jar_cmd, manifest, output_jar, cmd_file, srcjar_polluted, gen_file_map, include_base, pluck_base, files):
```

Comment:
do we still use this with the scala one? If not, can we remove? python not being hermetic worries me. bazelbuild/bazel#890